routers/api/v1/admin/hooks.go

```diff
@@ -74,7 +74,11 @@ func GetHook(ctx *context.APIContext) {
 	hookID := ctx.ParamsInt64(":id")
 	hook, err := webhook.GetSystemOrDefaultWebhook(ctx, hookID)
 	if err != nil {
-		ctx.Error(http.StatusInternalServerError, "GetSystemOrDefaultWebhook", err)
+		if webhook.IsErrWebhookNotExist(err) {
```

Comment:
```suggestion
		if errors.Is(err, util.ErrNotExist) {
```
additionally needs the corresponding imports

Reply:
fixed in https://github.com/go-gitea/gitea/pull/24823/commits/954abcd8c577976065bbab4cf3b6a23fcb740574